config/routes.rb

```diff
@@ -1,11 +1,19 @@
 Rails.application.routes.draw do
   root "drinks#index"
   devise_for :users, except: [:destroy]
+  resources :users, only: [:show]
 
   resources :drinks do
     resources :reviews, only: [:create, :index]
   end
 
+  namespace :admin do
+    get '/', :to => 'dashboards#show'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.